src/Liip/RMT/Action/BuildPharPackageAction.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/*
+ * This file is part of the project RMT
+ *
+ * Copyright (c) 2013, Liip AG, http://www.liip.ch
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\RMT\Action;
+
+use Phar;
+use FilesystemIterator;
+
+use Liip\RMT\Context;
+
+/**
+ * Builds a Phar package of the current project.
+ */
+class BuildPharPackageAction extends BaseAction
+{
+
+    protected $releaseVersion;
+
+    public function __construct($options)
+    {
+        $this->options = array_merge(array(
+            'package-name' => 'rmt-package',
+            'destination' => '/tmp/',
+            'excluded-paths' => '',
+            'metadata' => [],
```

Comment:
Personally I use it on two legacy projects that are still on 5.3... So as
long as it's not a big effort to keep that compatibility I prefer to keep it
Le 1 mai 2015 08:46, "Donato Rotunno" <notifications@github.com> a écrit :

> In src/Liip/RMT/Action/BuildPharPackageAction.php
> <https://github.com/liip/RMT/pull/95#discussion_r29491702>:
>
> > +
> > +/**
> > + * Builds a Phar package of the current project.
> > + */
> > +class BuildPharPackageAction extends BaseAction
> > +{
> > +
> > +    protected $releaseVersion;
> > +
> > +    public function __construct($options)
> > +    {
> > +        $this->options = array_merge(array(
> > +            'package-name' => 'rmt-package',
> > +            'destination' => '/tmp/',
> > +            'excluded-paths' => '',
> > +            'metadata' => [],
>
> You're right.
> But what about upping to PHP 5.4, especially considering
> http://php.net/supported-versions.php
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/liip/RMT/pull/95/files#r29491702>.
>